adapter-pulsar/src/main/java/io/nosqlbench/adapter/pulsar/dispensers/PulsarBaseOpDispenser.java

```diff
@@ -558,17 +517,12 @@ public Reader<?> getReader(String cycleTopicName,
                 //    startMsgId = MessageId.latest;
                 //}
 
-                reader = readerBuilder.startMessageId(startMsgId).create();
-
+                return readerBuilder.startMessageId(startMsgId).create();
             } catch (PulsarClientException ple) {
                 ple.printStackTrace();
```

Comment:
what about taking this chance of changing this code to remove "printStackTrace" and use the logger (and also pass the exception as the cause of the RuntimeException below ?